scripts/build.d/pybind11

```diff
@@ -10,20 +10,20 @@ if [ "${SRCSYNC}" == "tarball" ] ; then
   pkgver=${VERSION:-2.10.2}
   pkgfull=${pkgname}-${pkgver}
   pkgfn=${pkgfull}.tar.gz
-  pkgurl=https://github.com/pybind/pybind11/archive/refs/tags/v${pkgfull}
+  pkgurl=https://github.com/pybind/pybind11/archive/refs/tags/v${pkgver}.tar.gz
 
   download_md5 $pkgfn $pkgurl 695df9e99d92cdaa7fd0449fc5f30c26
 
   mkdir -p ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
   pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src > /dev/null
-    tar xf ${DEVENVDLROOT}/$pkgfn
+    tar xzf ${DEVENVDLROOT}/$pkgfn
```

Comment:
It's intentional to omit `z` or `j` so that tar can determine it for us.  Please do not add `z`.

Reply:
I see. Then, I will remove the other `z` in the vim build script I wrote in another PR.

Reply:
What is already there may remain there.  A good guideline for PR/commit is to minimize the diff.  When a maintainer does the archaeology in the future, he or she will appreciate the minimalism.